definitions/infra-awsdynamodbglobalsecondaryindex/summary_metrics.yml

```diff
@@ -3,17 +3,19 @@ providerAccountName:
     key: providerAccountName
   title: AWS account
   unit: STRING
-progress:
+consumedReadCapacityUnits:
+  goldenMetric: consumedReadCapacityUnits
   query:
```

Comment:
@mkrichevsky  I see the queries differ from the GM's. Would we need to add the 2 queries for awsSample and aws instrumentation provider?

Reply:
Mmh, probably the shim will help when resolving the summary metrics. All good then, I'll merge the PR.